### PR TITLE
Create 01_how-do-i-change-the-privacy-of-my-writings.md

### PR DESCRIPTION
**Pull request by DarklingMuse**
New folder and first PR for the FAQ writings section discussed in https://github.com/fetlife/help/pull/103/files

### CODE REVIEW

File: 18_writings/01_how-do-i-change-the-privacy-settings-on-writings [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# How do I change the privacy settings on my Writings?
```
Comment (nielsenramon) [inline 18_writings/01_how-do-i-change-the-privacy-settings-on-writings:1]: Make sure to rename the file to the same title as this one and add `.md` as file extension please

What do you think about:

`How do I change the privacy of my writings?`

File: 18_writings/01_how-do-i-change-the-privacy-settings-on-writings [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+You can change your privacy settings to 'Friends Only' or 'Kinksters' (everyone on Fetlife) at any time by:
```
Comment (nielsenramon) [inline 18_writings/01_how-do-i-change-the-privacy-settings-on-writings:3]: What about something like this:

```
There's 2 privacy settings for writings:

- FetLifers: Everyone on FetLife can read the writing
- Just Friends

You can change the privacy of a writing at any time by:
```

Just an idea! WDYT?

File: 18_writings/01_how-do-i-change-the-privacy-settings-on-writings [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+1. Opening the Writing that you want to change
```
Comment (nielsenramon) [inline 18_writings/01_how-do-i-change-the-privacy-settings-on-writings:5]: `1. Opening the writing`

File: 18_writings/01_how-do-i-change-the-privacy-settings-on-writings (hunk 2 above)
Comment (DarklingMuse) [reply 18_writings/01_how-do-i-change-the-privacy-settings-on-writings:3]: @nielsenramon I like that, breaks it down into less of a sentence and more quick nuggets of info

File: 18_writings/01_how-do-i-change-the-privacy-settings-on-writings (hunk 1 above)
Comment (DarklingMuse) [reply 18_writings/01_how-do-i-change-the-privacy-settings-on-writings]: @nielsenramon Renamed and updated with your suggestions. I was trying to keep the title to be similar to the original but that's much cleaner and clearer!

File: 18_writings/01_how-do-i-change-the-privacy-of-my-writings.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# How do I change the privacy of my Writings?
+
+There are 2 different privacy settings for Writings:
+
+- FetLifers (Everyone on FetLife can read the writing.)
```
Comment (nielsenramon) [inline 18_writings/01_how-do-i-change-the-privacy-of-my-writings.md:5]: We probably want to get rid of the capital E and the dot inside the brackets right? :D

**Review by nielsenramon — APPROVED** [review]
Apart from one small comment, good to go!

File: 18_writings/01_how-do-i-change-the-privacy-of-my-writings.md (hunk 4 above)
Comment (DarklingMuse) [reply 18_writings/01_how-do-i-change-the-privacy-of-my-writings.md:5]: @nielsenramon depends if you want it to be a run on sentence that is broken up i.e.

Fetlifers, everyone on fetlife can read the writing.

Or as an additional comment that is separate from the first statement but relates to it. (Asides are fun!)

Either way yes we need some punctuation - either a period on the end of FetLifers or removing the punctuation from inside the bracket

Comment (nielsenramon) [reply 18_writings/01_how-do-i-change-the-privacy-of-my-writings.md:5]: Maybe without brackets then? `FetLifers: everyone on FetLife can read the writing`

**Pull request merged**